src/inverseRobot.js

```diff
@@ -7,7 +7,22 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+  const uniqueValues = new Set(values);
+
+  if (uniqueValues.size !== values.length) {
+    return null;
+  }
+
+  const invertedRobot = {};
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    invertedRobot[value] = key;
```

Comment:
in this case, you could've avoided using variable
why? you use this value once, so there's no actual purpose